cpp/src/io/parquet/page_string_decode.cu

```diff
@@ -608,8 +608,8 @@ CUDF_KERNEL void __launch_bounds__(preprocess_block_size)
   if (t == 0) {
     // don't clobber these if they're already computed from the index
     if (!pp->has_page_index) {
-      s->page.num_nulls  = 0;
-      s->page.num_valids = 0;
+      pp->num_nulls  = 0;
```

Comment:
`s` is not initialized at this point so directly update `pp` (PageInfo) whose reference `s` will store in `setup_local_page_info` function later